packages/hierarchies/src/presentation-hierarchies.ts

```diff
@@ -21,7 +21,18 @@ export {
 export { RowsLimitExceededError } from "./hierarchies/HierarchyErrors";
 export { GroupingHierarchyNode, HierarchyNode, NonGroupingHierarchyNode, ParsedHierarchyNode, ProcessedHierarchyNode } from "./hierarchies/HierarchyNode";
 export { HierarchyNodeIdentifier, HierarchyNodeIdentifiersPath } from "./hierarchies/HierarchyNodeIdentifier";
-export { GroupingNodeKey, HierarchyNodeKey, InstancesNodeKey } from "./hierarchies/HierarchyNodeKey";
+export {
+  GroupingNodeKey,
+  HierarchyNodeKey,
+  InstancesNodeKey,
+  ClassGroupingNodeKey,
+  LabelGroupingNodeKey,
+  PropertyGroupingNodeKey,
+  PropertyOtherValuesGroupingNodeKey,
+  PropertyValueGroupingNodeKey,
+  PropertyValueRangeGroupingNodeKey,
+  StandardHierarchyNodeKey,
```

Comment:
Do you think we need to export standard node key?

Reply:
I think we do, since we have `HiearchyNodeKey.isStandard()` function.